background/main.js

```diff
@@ -1,11 +1,22 @@
 var SERVER_URL = "https://chronicle.dev.mozaws.net";
```

Comment:
it would be nice to fetch this value from a user-configurable pref in the extension, maybe we can file a bug or add a todo for that